docs/developer/app-store/apps/avatax/configuration.mdx

```diff
@@ -171,3 +171,17 @@ The invoice will have line items from the Saleor order. Clicking on a line item
 Clicking **"Review this calculation"** will show more details about taxes calculation including custom rules, tax type, address used for calculation, exceptions applied, etc.:
 
 ![Review the calculation page with information about tax details for the particular line item in order](./avatax-details-2.png)
+
+## Best practices
+
+### Learn how to setup multiple tax configurations
+
+If your use case requires you to have multiple AvaTax configurations, create multiple providers within a single AvaTax app. You can then assign a provider to the relevant channels. Avoid installing a second AvaTax app.
+
+### Don't request tax fields if they are not needed
+
+If your application or integration doesn't need the `TaxedMoney` type field, don't add it to the GraphQL subscription or query. This way, Saleor won't send webhook requests to the AvaTax app for unnecessary tax calculations, making your integration faster.
```

Comment:
Would also add: 

An example of such an approach would be skipping taxes information in "Cart" views (and mutations modifying it's contents) and only requesting such information during checkout process.

WDYT?

Reply:
Awesome idea - added 👍🏻